manual/manual_en.tex

```diff
@@ -259,17 +259,17 @@ \section{History of FidoCadJ}
 a new program, rewritten from scratch.
 
 The choice of using Java\index{Java} is due to the fact that I currently use many different operating systems. Spending time
-and energies on something that is not completely portable does not
+and energy on something that is not completely portable does not
 appeal to me anymore. The effort of learning the Cocoa framework would
 have probably given a better result on macOS, but it would have
 made FidoCadJ completely non-portable. I am not a computer guy. The
 time I spend to program is time taken away from my electronics interests.
 In fact, especially at the beginning, the FidoCadJ code source showed that
 I am not a very Java\index{Java} and object oriented programming\index{Object oriented programming}
 purist and several solutions were
-more practical than elegant. However, I tried to improve continuously the overall code quality FidoCadJ has been quite successfully compared to other Open Source projects at least in one occasion\footnote{See for example \href{http://www.cs.usask.ca/~croy/papers/2013/RahmanStackOverflowTechReport.pdf}{Rahman, M. M., Roy, C. K., and Keivanloo, I. ``Subjective Evaluation of Software Quality Using Crowdsource Knowledge: An Exploratory Study''. Univ. of Saskatchewan, Department of Computer Science technical report 2013-01}}. Continuing in this way, static and dynamic analysis of the source code has been exploited to suggest where to refactor and improve the code organization.
+more practical than elegant. However, I tried to improve continuously the overall code quality FidoCadJ.  It has been quite successfully compared to other Open Source projects at least in one occasion\footnote{See for example \href{http://www.cs.usask.ca/~croy/papers/2013/RahmanStackOverflowTechReport.pdf}{Rahman, M. M., Roy, C. K., and Keivanloo, I. ``Subjective Evaluation of Software Quality Using Crowdsource Knowledge: An Exploratory Study''. Univ. of Saskatchewan, Department of Computer Science technical report 2013-01}}. Continuing in this way, static and dynamic analysis of the source code has been utilized to suggest where to refactor and improve the code organization.
```

Comment:
Would the following sentence work in this context?
"However, I tried to improve continuously the overall code quality. FidoCadJ has been quite successfully compared..."

Reply:
Ok, thank you again very much!